v2/bench_test.go

```diff
@@ -42,7 +41,9 @@ func BenchmarkReadBlocks(b *testing.B) {
 				}
 			}
 
-			cr.Close()
+			if err := r.Close(); err != nil {
+				b.Fatal(err)
+			}
```

Comment:
FYI, no need to check for an error, since we just read

Reply:
Trying to be tidy, since close _can_ return an error. might be an overkill here agreed.